scripts/scenario_31_payload.lua

```diff
@@ -658,7 +654,7 @@ function SpawnWave(faction)
       if i == queenIndex and WaveSize > 1 then
         enemyTemplate = "Phobos M3"
       end
-      local enemy = CpuShip():setTemplate(enemyTemplate):setFaction("Kraylor"):setPosition(enemyX, enemyY)
+      local enemy = CpuShip():setTemplate(enemyTemplate):setFaction("Kraylor"):setPosition(enemyX, enemyY):orderIdle()
```

Comment:
That idle isn't the default should be a bug.

Reply:
Noted. I'll see if I can implement a fix on the engine side, but this may also fix another edge case for me so might as well stay.